web/src/main/webapp/WEB-INF/views/viewAllCourses.jsp

```diff
@@ -0,0 +1,231 @@
+<script async="" src="javascript/analytics.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<style id="style-1-cropbar-clipper">
+    .en-markup-crop-options {
+        top: 18px !important;
+        left: 50% !important;
+        margin-left: -100px !important;
+        width: 200px !important;
+        border: 2px rgba(255, 255, 255, .38) solid !important;
+        border-radius: 4px !important;
+    }
+
+    .en-markup-crop-options div div:first-of-type {
+        margin-left: 0px !important;
+    }
+
+    .colortext {
+        background-color: #ffe; /* Цвет фона */
+        color: black; /* Цвет текста */
+    }
+
+    .buttonText {
+        color: black;
+    }
+</style>
+</head>
+<body class="inside footer-under">
+<script type="text/javascript">
+    $(document).ready(function () {
+        $('#perPage').change(function () {
+            $('#submit').click();
+        });
+
+        $('a.page').click(function () {
+            var id = $(this).attr('id');
+            $('#nextPage').val(id);
+            $('#submit').click();
+        });
+    });
+</script>
+<noscript>&lt;div&gt;&lt;img src="https://mc.yandex.ru/watch/37563830" style="position:absolute; left:-9999px;" alt="" /&gt;&lt;/div&gt;</noscript>
+<div class="nav navbar navbar-fixed-top navbar-hide">
+    <div class="container"><a href="https://vertex-academy.com/index.html" class="logo pull-left"></a>
+        <button data-toggle="collapse" data-target=".navbar-collapse" class="navbar-toggle"><span
+                class="glyphicon glyphicon-align-justify"></span></button>
+        <div class="collapse navbar-collapse navbar-responsive-collapse">
+            <ul class="navbar-nav">
+                <li><a href="https://vertex-academy.com/index.html#courses">Курсы</a></li>
+                <li><a href="https://vertex-academy.com/index.html#about">о нас</a></li>
+                <li><a href="https://vertex-academy.com/index.html#faq">FAQ</a></li>
+                <li><a href="https://vertex-academy.com/index.html#feedback">Отзывы</a></li>
+                <li><a href="https://vertex-academy.com/index.html#lecturers">Преподаватели</a></li>
+                <li><a href="https://vertex-academy.com/tutorials/index.php" class="link">Блог</a></li>
+                <li><a href="https://vertex-academy.com/index.html#contacts">Контакты</a></li>
+                <li><a href="https://vertex-academy.com/tutorials/en/login/">Войти </a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+<div class="page gray-page mh100">
+    <div class="container pt1_5">
+        <div align="center">
+            <br/>
+            <br/>
+            <br/>
+            <a href="<c:url value="/addCourse"/>">Add course</a>
+            <br/>
+            <br/>
+            <br/>
+            <c:if test="${dataNavigator.currentNumberPage!=1}">
+                <a id="1" class="page" style="cursor: pointer;"><<</a>
```

Comment:
This is not strange. This is the << link to the first page, and this >> to the last page